app/workers/education_form/process_10203_submissions.rb

```diff
@@ -0,0 +1,164 @@
+    def process_user_submissions(user_submissions)
+      user_submissions.each do |user_uuid, submissions|
+        gi_bill_status = get_gi_bill_status(user_uuid)
+        if gi_bill_status == {} || gi_bill_status.remaining_entitlement.blank?
+          submissions.each { |submission| update_status(submission, DENIED) }
+        elsif submissions.count > 1
+          check_previous_submissions(submissions, gi_bill_status)
+        else
+          process_submission(submissions.first, gi_bill_status)
+        end
+      end
+    end
+
+    # Retrieve EVSS gi_bill_status data for a user
+    def get_gi_bill_status(user_uuid)
+      user = User.find(user_uuid)
+      service = EVSS::GiBillStatus::Service.new(user)
+      service.get_gi_bill_status
+    rescue => e
+      Rails.logger.error "Failed to retrieve GiBillStatus data: #{e.message}"
+      {}
+    end
+
+    def update_status(submission, status)
+      submission.education_stem_automated_decision.update(automated_decision_state: status)
+    end
+
+    # Makes a list of all submissions that have not been processed and have a status of INIT
+    # Finds most recent submission that has already been processed
+    #
+    # Submissions are marked as processed in EducationForm::CreateDailySpoolFiles
+    #
+    # For each unprocessed submission compare benefit_left and pursuing_teaching_cert values
+    #     to most recent processed submissions
+    # If values are the same set status as PROCESSED
+    # Otherwise check submission data and EVSS data to see if submission can be marked as PROCESSED
+    def check_previous_submissions(submissions, gi_bill_status)
+      unprocessed_submissions = submissions.find_all do |ebc|
+        ebc.processed_at.nil? && ebc.education_stem_automated_decision&.automated_decision_state == INIT
+      end
+      most_recent_processed = submissions.find_all do |ebc|
+        ebc.processed_at.present? && ebc.education_stem_automated_decision&.automated_decision_state != INIT
+      end
+                                         .max_by(&:processed_at)
+
+      processed_form = format_application(most_recent_processed)
+
+      unprocessed_submissions.each do |submission|
+        unprocessed_form = format_application(submission)
+        if unprocessed_form.benefit_left == processed_form.benefit_left &&
+           unprocessed_form.pursuing_teaching_cert == processed_form.pursuing_teaching_cert
+          update_status(submission, PROCESSED)
+        else
+          process_submission(submission, gi_bill_status)
+        end
+      end
+    end
+
+    # Set status to DENIED when pursing_teaching_cert in form data is 'no' (false)
+    #   or EVSS data for a user shows there is more than 6 months of remaining_entitlement
+    def process_submission(submission, gi_bill_status)
+      submission_form = format_application(submission)
+      status = if !submission_form.pursuing_teaching_cert || more_than_six_months?(gi_bill_status)
+                 DENIED
```

Comment:
Based on AC5a, this should be 
`if !(submission_form.pursuing_teaching_cert || submission.is_enrolled_stem) || more_than_six_months?(gi_bill_status)
  DENIED`

Reply:
fixed